mycroft/skills/settings.py

```
@@ -57,17 +57,207 @@
+
+import yaml
 from ovos_backend_client.api import DeviceApi
 from ovos_backend_client.pairing import is_paired, requires_backend
-from ovos_utils.log import LOG
 from ovos_backend_client.settings import RemoteSkillSettings, get_display_name
+
 from ovos_bus_client.message import Message, dig_for_message
+from ovos_utils.log import LOG
 
 # backwards compat imports
 from mycroft.deprecated.skills.settings import SkillSettingsDownloader, SettingsMetaUploader, load_remote_settings_cache, \
```

Ruff

`mycroft.deprecated.skills.settings.SkillSettingsDownloader` imported but unused

Remove unused import

(F401)